script/release.js

```diff
@@ -8,12 +8,12 @@ const currentBranch = (spawnOrFail('git', [' branch --show-current'], { skipOutp
 const deployDemo = (version) => {
   const formattedVersion = `${version.replace(/\./g, "-")}`;
   process.chdir(path.join(__dirname, '../demos/serverless'));
-  const meetingDemoName = `chime-sdk-demo-${formattedVersion}`;
+  const meetingDemoName = `chime-sdk-demo-global-${formattedVersion}`;
   logger.log(`Deploying ${meetingDemoName} ...`);
-  spawnOrFail('npm', [`run deploy -- -b ${meetingDemoName} -s ${meetingDemoName} -o ${meetingDemoName} -u false`], { printErr: true });
+  spawnOrFail('npm', [`run deploy -- -b ${meetingDemoName} -s ${meetingDemoName} -o chime-sdk-demo-global -u false`], { printErr: true });
```

Comment:
Why are we changing the -o param?

Reply:
So we can test media capture in both global and regional demo.  Also we change it to a fix name so we do not create new S3 buckets every time for each release.

Reply:
Gotcha, I misunderstood what the `-o` option was doing